lib/prototype-admin/clear-data.html

```diff
@@ -21,8 +21,9 @@
       <h1 class="govuk-heading-xl">
         Clear data?
       </h1>
+      <p>This will clear data all session data and reload the default session data.
```

Comment:
This is missing a closing paragraph tag, and has a typo ('…will clear data all session data…')

How about…

```suggestion
      <p>This will clear all of the data entered in this session, and reload any default values.</p>
```